Documentation/distributions.md

```diff
@@ -0,0 +1,50 @@
+# How to get rkt on each Linux distribution
+
+## CoreOS
+
+rkt is pre-installed on CoreOS.
+The [CoreOS release page](https://coreos.com/releases/) specifies the version of rkt available in each CoreOS release channel.
```

Comment:
Nit: It would be nice to have a new document use reference-style links in its markdown source. I think you may find it easier to write, even: https://github.com/coreos/docs/blob/master/STYLE.md#marking-down-the-link. So the link in this line might become, e.g.:
`[CoreOS releases page][coreos-releases]`

and before EOF that label is declared and defined:
`[coreos-releases]: {{site.baseurl}}/releases/`